static/lang/re-en.json

```diff
@@ -4738,6 +4738,7 @@
                 "SkillFeat": "Select a skill feat.",
                 "SkillTrained": "Select a trained skill.",
                 "SkillUntrained": "Select an untrained skill.",
+                "Speed": "Select a Speed.",
```

Comment:
```suggestion
```
Turns out we have `PF2E.SpecificRule.Prompt.MovementType`. You can use that instead!